cli/cmd/auth.go

```diff
@@ -180,6 +181,10 @@ func verifyAuthParams(authParams *authCmdParams) error {
 				return fmt.Errorf("Error in fetching the api-token\n" + err.Error() + "\nCLI is not authenticated")
 			}
 		}
+		namespace, err = smartKeptnCLIAuth()
+		if err != nil {
+			return err
+		}
```

Comment:
The smartKeptnCLIAuth() check is done after the other checks are completed as proposed in https://github.com/keptn/keptn/issues/2877#issuecomment-770708493